CanvasPlusPlayground/Features/Files/FileViewer.swift

```diff
@@ -26,7 +30,13 @@ struct FileViewer: View {
                     .toolbar(.hidden)
                     #else
                     .toolbar {
-                        ShareLink(item: url)
+                        ToolbarItemGroup {
+                            ShareLink(item: url)
+                            DownloadButton(url: url, fileName: file.displayName)
+                            Button("Open", systemImage: "doc.text.magnifyingglass") {
```

Comment:
I think this symbol is more appropriate:
```suggestion
                            Button("Open", systemImage: "arrow.up.forward.app") {
```